bom-latest/pom.xml

```diff
@@ -282,6 +298,31 @@
                 <artifactId>jquery-detached</artifactId>
                 <version>1.2.1</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkinsci.plugins</groupId>
+                <artifactId>pipeline-model-api</artifactId>
+                <version>${pipeline-model-definition-plugin.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkinsci.plugins</groupId>
+                <artifactId>pipeline-model-declarative-agent</artifactId>
```

Comment:
This plugin is deprecated. Is the dep coming from somewhere?

Reply:
think it was used by an older version of pipeline-model-definition, so removed it